packages/account/src/Components/poi/status/unsupported/card-details/uploader.tsx

```diff
@@ -101,17 +101,17 @@ const Uploader = ({ data, value, is_full, onChange, has_frame }: Partial<TUpload
         setImage(value);
     }, [value]);
 
-    const handleChange = (file?: object, setFieldValue?: FormikProps<FormikValues>['setFieldValue']) => {
+    const handleChange = (file?: object | null, setFieldValue?: FormikProps<FormikValues>['setFieldValue']) => {
```

Comment:
We should make these params mandatory to prevent these kind of conditions. It got missed as the params are optional so there wasn't any type error for arguments missing